scripts/travis_install_mcore.sh

```diff
@@ -0,0 +1,4 @@
+git clone https://github.com/trailofbits/manticore.git
+cd manticore
+pip install --no-binary keystone-engine -e .[dev]
```

Comment:
we probably don't need to do a dev install, since we're not running mcore tests here? rm'ing this will let us remove the unicorn line in the travis yml too